pkg/hintrunner/zero/zerohint_ec_test.go

```diff
@@ -1147,6 +1147,30 @@ func TestZeroHintEc(t *testing.T) {
 				}),
 			},
 		},
+		"GetHighLen": {
+			{
+				operanders: []*hintOperander{
+					{Name: "len_hi", Kind: uninitialized},
+					{Name: "scalar_u.d2", Kind: apRelative, Value: feltString("10")}, 
```

Comment:
You kind of need to modify your unittests, so that they mock the `BigInt3` struct in the memory. Here is the reference on how to implement `BigInt3` in our unittest suite:
https://github.com/NethermindEth/cairo-vm-go/blob/a55dae8d65d234b6e5a6b2d53cfbfb3d096c377d/pkg/hintrunner/zero/zerohint_ec_test.go#L17